src/Components/shareList.js

```diff
@@ -1,4 +1,6 @@
 import React, { useState } from 'react';
+import '../App.css';
```

Comment:
I generally like to [organize my imports](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/order.md) so as you maintain this file, it's easy to understand the types of dependencies it has on other parts of the codebase. Sometimes a file has a lot of third-party dependencies, or maybe local dependencies. Organizing them in groups like this will help you make sense of that easily!